scaleway/resource_registry_namespace_beta_test.go

```diff
@@ -0,0 +1,98 @@
+package scaleway
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/scaleway/scaleway-sdk-go/api/registry/v1"
+)
+
+func TestRegistryNamespaceBeta(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckScalewayRegistryNamespaceBetaDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: `
+					resource scaleway_registry_namespace_beta cr01 {
+						name = "test-cr"
+					}
+				`,
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckScalewayRegistryNamespaceBetaExists("scaleway_registry_namespace_beta.cr01"),
```

Comment:
Remove empty line

Reply:
To which empty line are you referring to?

Reply:
Sorry, it was a diff issue.